cli/tests/test_git_colocated.rs

```diff
@@ -145,7 +120,6 @@ fn test_git_colocated_unborn_bookmark() {
     ◆  0000000000000000000000000000000000000000
     "###);
     // Staged change shouldn't persist.
-    checkout_index();
```

Comment:
We need to either check out the index content to working copy or assert the index contents to ensure that the Git index is reset by `jj new root()`.